tests/test_views.py

```diff
@@ -519,6 +523,95 @@ async def test_notifications_with_disabled_option(
     )
     assert resp.status == HTTPStatus.FORBIDDEN
 
+async def test_notifications_with_no_expiration(
+    hass_client_local_frigate: Any,
+    hass: Any,
+) -> None:
+    """Test that notification events are served if they are set to not expire."""
+
+    # Make another config entry with the same data but with
+    # CONF_NOTIFICATION_PROXY_ENABLE disabled.
+    private_config_entry = create_mock_frigate_config_entry(
+        hass,
+        entry_id="private_id",
+        options={CONF_EXPIRE_NOTIFICATIONS_AFTER_MINS: 0},
+        data=hass.config_entries.async_get_entry(TEST_CONFIG_ENTRY_ID).data,
+    )
+
+    private_config: dict[str, Any] = copy.deepcopy(TEST_CONFIG)
+    private_config[ATTR_MQTT][ATTR_CLIENT_ID] = "private_id"
+    private_client = create_mock_frigate_client()
+    private_client.async_get_config = AsyncMock(return_value=private_config)
+
+    await setup_mock_frigate_config_entry(
+        hass, config_entry=private_config_entry, client=private_client
+    )
+
+    # Fake time is 2021-11-01T19:02:00
+    with patch("custom_components.frigate.views.NotificationsProxyView._get_current_datetime",
+    return_value=datetime(2021, 11, 1, 19, 2, 00, 000000)):
+
+        # Old event id should be vended
+        # Test event timestamp is 2020-01-01 00:00:00
+        resp = await hass_client_local_frigate.get(
+            f"/api/frigate/private_id/notifications/1577854800.123456-random/snapshot.jpg"
+        )
+        assert resp.status == HTTPStatus.OK
+
```

Comment:
Nit: Think you may need  to format this with  `black` and/or run the pre-commit, as I would have expected  two blank lines here.

Reply:
Will do thank you. We could use adding a CONTRIBUTING.md file with instructions on how to get a dev environment set up and a checklist of things to run prior to publishing a PR.